### PR TITLE
Create initial admin RBAC for CNV-QE

### PR DESCRIPTION
**Pull request by sarahbx**
Adding CNV-QE to services. Will add additional admins after they have joined the openshift org.
@stevekuznetsov

### CODE REVIEW

File: services/cnv-qe/admin_rbac.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+metadata:
+  name: cnv-qe-image-puller-binding
+  namespace: ocp
+roleRef:
+  kind: ClusterRole
+  apiGroup: rbac.authorization.k8s.io
+  name: system:image-puller
+subjects:
+- kind: ServiceAccount
+  namespace: ocp
```
Comment (alvaroaleman) [inline services/cnv-qe/admin_rbac.yaml:26]: I think you want that to point to `cnv-qe`

File: services/cnv-qe/admin_rbac.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  name: image-puller
+---
+kind: Group
+apiVersion: v1
```
Comment (alvaroaleman) [inline services/cnv-qe/admin_rbac.yaml:30]: please use `user.openshift.io/v1`

File: services/cnv-qe/OWNERS [hunk 3]
```diff
@@ -0,0 +1,2 @@
+approvers:
+- sarahbx
```
Comment (alvaroaleman) [inline services/cnv-qe/OWNERS:2]: Are you the only member of this team?

Comment (sarahbx) [reply services/cnv-qe/OWNERS:2]: I am not, but others are not part of openshift org yet. I have provided them instructions, and plan to add them afterwards.

File: services/cnv-qe/admin_rbac.yaml (hunk 1 above)
Comment (sarahbx) [reply services/cnv-qe/admin_rbac.yaml:26]: yes, missed one. thank you.

**Review by alvaroaleman — APPROVED** [review]
/label tide/merge-method-squash

**Pull request merged**